### DISCUSSION TITLE
Im getting an error when I try to access the Solidity by example website ...

### DISCUSSION THREAD

**Discussion by LaserLikeBucko** [Cyfrin/foundry-full-course-cu#1632] [Q&A]
Hi there.  Im on the fund me lesson in Patricks superb course and I have hit a minor roadblock when I try to access the https://solidity-by-example.org site I get the following error. 
'''
Secure Connection Failed

An error occurred during a connection to solidity-by-example.org. Peer received a valid certificate, but access was denied.

Error code: SSL_ERROR_ACCESS_DENIED_ALERT

    The page you are trying to view cannot be shown because the authenticity of the received data could not be verified.
    Please contact the web site owners to inform them of this problem.
'''
I'm just wondering is it me or is it them?  The error seems to suggest them - is three anything I can do - it looks like a very valuable resource and I suffer from FOMO
Thanks in advance for any help offered.
wr
M

**Comment by EngrPips** [comment]
Hello @LaserLikeBucko, It seems the problem is from your end. I tried to visit the site, and it loaded for me. 

<img width="1440" alt="Screenshot 2024-04-22 at 17 17 19" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/95760918/b53349ff-e289-4499-96c5-28213e4f35d9">

Try a different browser from the browser you are currently using to open the website and see if it works for you. Mind you, I used Chrome.

**Comment by drained69** [comment]
Try changing your network source or using a VPN, helps sometimes.

**Comment by c0deg33k** [comment]
Hello @LaserLikeBucko,

The error you're encountering, ```SSL_ERROR_ACCESS_DENIED_ALERT```, typically indicates a problem with the ```SSL/TLS``` certificate of the website. This can happen for a variety of reasons, such as the certificate being expired, not trusted by your browser, or the server configuration not allowing access from your location or device.

Here are a few steps you can take to troubleshoot and potentially resolve this issue:

- Check the Date and Time on Your Device
- Update Your Browser
- Clear Browser Cache and Cookie
- Try a Different Browser or Device
- Check for Browser Extensions that may interfere with SSL certificates
